rfcs/20190729-keras-preprocessing.md

```diff
@@ -0,0 +1,738 @@
+# Keras Preprocessing API
+
+| Status        | Proposed      |
+:-------------- |:---------------------------------------------------- |
+| **Author(s)** | Francois Chollet (fchollet@google.com)               |
+| **Updated**   | 2019-07-29                                           |
+
+
+## Context
+
+`tf.data.Dataset` is the main API for data loading and preprocessing in TensorFLow. It has two advantages:
+
+- It supports GPU prefetching
```

Comment:
"It supports GPU prefetching" is a small subset of the following advantages:

- it supports parallelization of transformations
- it supports parallelization of I/O
- it supports software pipelining (e.g. prefetching to GPU / TPU)

Reply:
I think the firsts two advantages you mentioned are not there because they are already supported by the current API.